integration-cli/docker_cli_proxy_test.go

```diff
@@ -40,24 +38,15 @@ func (s *DockerDaemonSuite) TestCliProxyProxyTCPSock(c *check.C) {
 		}
 	}
 
-	if ip == "" {
-		c.Fatal("could not find ip to connect to")
-	}
-
-	if err := s.d.Start("-H", "tcp://"+ip+":2375"); err != nil {
-		c.Fatal(err)
-	}
-
+	c.Assert(ip, checker.Equals, "", "could not find ip to connect to")
```

Comment:
not sure what will happen to the fourth arg here. It can just be a regular comment on that line. `// could not find ip to connect to`. Gocheck will print the whole line if the assert fails.